adapters/ix/ix.go

```diff
@@ -417,6 +419,34 @@ func (a *IxAdapter) MakeBids(internalRequest *openrtb2.BidRequest, externalReque
 				}
 			}
 
+			var bidNative *native1response.Response
+			if bidType == openrtb_ext.BidTypeNative {
```

Comment:
Please add JSON tests for this new change

Reply:
Added!